OpenRA.Mods.Common/Activities/UnloadCargo.cs

```diff
@@ -125,6 +125,8 @@ public override bool Tick(Actor self)
 					pos.SetVisualPosition(actor, spawn);
 
 					actor.CancelActivity();
+					actor.QueueActivity(move.MoveIntoWorld(actor));
```

Comment:
This would make the actor move into the world twice when loaded via `RideTransport`.

Reply:
It does, but a) doesn't appear to regress anything and b) shouldn't regress anything. `MoveIntoWorld` should be smart enough to handle this (if a unit is already at mobile.ToSubCell and mobile.ToCell).